tensorrt_llm/evaluate/audio_asr.py

```
@@ -0,0 +1,515 @@
+def _compute_wer(
+    predictions: list[str],
+    references: list[str],
+    sample_ids: Optional[list[str]] = None,
+) -> tuple[float, list[ASRWERSampleResult]]:
+    if sample_ids is None:
+        sample_ids = [str(idx) for idx in range(len(predictions))]
+
+    sample_results = []
+    total_edits = 0
+    total_words = 0
+    for sample_id, prediction, reference in zip(sample_ids, predictions, references):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)